### PR TITLE
Minor fixes as render stream table mechanics and such

### PR DESCRIPTION
**Pull request by karlowski**
Fixed stream table render via .map without adding dumb x20 lines manually. Also URL's and such things like client ID broken into separate constants and *smart* components pulled up on App.js level.

### CODE REVIEW

File: src/components/streams.js [hunk 1]
```diff
@@ -1,60 +1,51 @@
...
+  const renderStreamTable = () => {
+    return streams.map((stream, i) => {
+      return (
+        <div className="stream-wrapper">
+          {renderSingleStream(i)}
+        </div>
+      )
+    })
+  }
+
```
Comment (FominSergiy) [inline src/components/streams.js:45]: nice work! Great use of .map() to parse the array and use of sub functions to parse each stream :)

File: src/components/streams.js [hunk 2]
```diff
@@ -1,60 +1,51 @@
...
-      <div className="stream-wrapper">{renderSingleStream(11)}</div>
-      <div className="stream-wrapper">{renderSingleStream(12)}</div>
-      <div className="stream-wrapper">{renderSingleStream(13)}</div>
-      <div className="stream-wrapper">{renderSingleStream(14)}</div>
-      <div className="stream-wrapper">{renderSingleStream(15)}</div>
-      <div className="stream-wrapper">{renderSingleStream(16)}</div>
-      <div className="stream-wrapper">{renderSingleStream(17)}</div>
-      <div className="stream-wrapper">{renderSingleStream(18)}</div>
-      <div className="stream-wrapper">{renderSingleStream(19)}</div>
+      {renderStreamTable()}
```
Comment (FominSergiy) [inline src/components/streams.js:48]: simplicity rules!

**Review by FominSergiy — COMMENTED** [review]
great work!

File: src/components/streams.js (hunk 2 above)
Comment (karlowski) [reply src/components/streams.js:48]: yep!

**Pull request merged**